src/components/SortedList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ListItem from './ListItem';
+
+const SortedList = ({ itemsKeys, data }) => {
+  return (
+    <ul>
+      {itemsKeys
+        // a stolen line from Andrew's blog
```

Comment:
Hahaha, Nice! i guess we have to thank Andrew for giving us an efficient method to sort. Great work implementing it here